packages/get-safe-property-chain/package.json

```diff
@@ -24,6 +24,9 @@
     "build": "tsc && tsc --project tsconfig.esm.json",
     "build:watch": "tsc --watch",
     "lint": "eslint src",
+    "postbuild": "npm pack",
+    "postinstall": "npm run build",
```

Comment:
Lerna says that it invokes, `prepublish`, and `prepare` commands after a CI, but the problem with those commands is that `prepublish` is deprecated, and both of them are also invoked by npm before the `npm pack` command. So i chose to use `postinstall` and `prepublishOnly`.